NOSQL/Assignment_doument_structure.txt

```diff
@@ -0,0 +1,37 @@
+Users
+{
+	id:1
+	user_name:"abc",
+	password:"zbxc123"
+	name:"anc asd",
+	date_of_birth:12-12-1990,
+	gender:"m or f"
+}
+
+Posts
+{
+	author_id:1
+	title:"post_title",
+	date_of_post:"11-2-2016 10:30",
+	content:{images,etc},
```

Comment:
If this is sub document, then you should specify the keys of the sub document. 
If this is a string, you should specify "" instead of {}